published_bucket.tf

```diff
@@ -14,126 +14,22 @@ data "aws_iam_role" "aws_config" {
   name = "aws_config"
 }
 
-#Create policy document
```

Comment:
If you've already deployed this bucket policy beyond Dev & Management Dev then you're going to need an intermediate PR here. First PR replaces the policy with something like the below, then second PR removes the policy entirely.

  statement {
     sid    = "Enable root-delegated access control"
     effect = "Allow"

     principals {
       type        = "AWS"
       identifiers = ["arn:aws:iam::${local.account[local.environment]}:root"]
     }

     actions   = ["kms:*"]
     resources = ["*"]
   }